tests/tests/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# Test source files
+file(GLOB SOURCES_SC2LADDERSERVER_TESTS "*.cpp" "*.h")
+
+# copy test data/configs
+configure_file(${CMAKE_CURRENT_LIST_DIR}/test_configs/TestMatch_Bot1Eliminated/LadderBots.json
+    ${EXECUTABLE_OUTPUT_PATH}/test_configs/TestMatch_Bot1Eliminated/LadderBots.json COPYONLY)
+configure_file(${CMAKE_CURRENT_LIST_DIR}/test_configs/TestMatch_Bot1Eliminated/LadderManager.conf
+    ${EXECUTABLE_OUTPUT_PATH}/test_configs/TestMatch_Bot1Eliminated/LadderManager.conf COPYONLY)
+
+# Include directories
+include_directories(SYSTEM
+    ${PROJECT_SOURCE_DIR}/tests/tests
```

Comment:
The original to support the following structure:
tests/unitests/testa
tests/unittests/testb
tests/functional/debugbot

tests/tests looks misleading to me.